Users.md

```diff
@@ -0,0 +1,111 @@
+**T-Mobile**  
+Use Case: Identity: Role-based Access Control  
+Project Link: https://opensource.t-mobile.com/blog/posts/hyperdirectory-blockchain-innovation/  
+Code: https://github.com/hyperledger/sawtooth-next-directory  
+Sawtooth Quote: “As the Un-carrier, T-Mobile approaches the wireless industry differently, from the customer experience to pushing the limits of the technologies that power our business and service,” said Warren McNeel, Senior Vice President, Digital Technology and Development, T-Mobile. “Using Hyperledger Sawtooth as a platform, the company created Sawtooth Hyper Directory as an Identity and Access Management (IAM) solution, and we’re excited to continue to develop new solutions on the platform that can help add value and functionality for both customers and our business.”  
+Contact:  
+
+**Tel Aviv Stock Exchange**  
+Use Case: Securities Lending  
+Project Link: https://www.ledgerinsights.com/tel-aviv-stock-exchange-blockchain/  
+Code:  
```

Comment:
Is the intention for these particular users to fill in these blanks?

Reply:
Yep! Once we get this merged I'm going to reach out to the contacts I know to have them update it.